### ISSUE TITLE
Cannot find/clone url

### ISSUE THREAD

**Issue by hginzel** [desktop/desktop#9565]
### Describe the feature or problem you’d like to solve

A clear and concise description of what the feature or problem is.

### Proposed solution

How will it benefit Desktop and its users?

### Additional context

Add any other context like screenshots or mockups are helpful, if applicable.

**Comment by rafeca** [comment]
Thanks for filing this issue @hginzel !

The `GitHub.com` tab on the Clone repository dialog allows you to clone your GitHub repositories  (the ones that appear in https://github.com/hginzel?tab=repositories).

To be able to clone any other repository you need to use the `URL` tab, where you can type any GitHub repository by `name/repo`.

This is indeed a bitconfusing, since one would expect that the `GitHub.com`  tab would allow to clone any GitHub repository.

I think that some tweaks to the clone UI can make it less confusing, for example we could allow to clone any GitHub repository from the GitHub.com tab (while only allowing the smart filtering for your own repositories).

@billygriffin @ampinsk any thoughts about this?

**Comment by ampinsk** [comment]
@rafeca I totally agree with your assessment here! This distinction we make is confusing and feels unnecessary. I'm not sure off the top of my head what the best way to go is, but agree tweaking the search on the GitHub.com tab to allow for this use case could be a helpful improvement, but maybe even combining these tabs could work. 🤔
